web/crux-ui/src/components/main/sidebar.tsx

```diff
@@ -21,85 +21,99 @@ import Link from 'next/link'
 import NavButton from './nav-button'
 import { MenuOption, NavSection } from './nav-section'
 
+export type MenuSection = {
+  title: string
+  items: MenuOption[]
+}
+
 export interface SidebarProps {
   className?: string
 }
 
+export const SIDEBAR_SECTIONS: MenuSection[] = [
+  {
+    title: 'project',
+    items: [
+      {
+        icon: '/projects.svg',
+        text: 'projects',
+        link: ROUTE_PROJECTS,
+      },
+      {
+        icon: '/deploy.svg',
+        text: 'deployments',
+        link: ROUTE_DEPLOYMENTS,
+      },
+    ],
+  },
+  {
+    title: 'components',
+    items: [
+      {
+        icon: '/servers.svg',
+        text: 'nodes',
+        link: ROUTE_NODES,
+      },
+      {
+        icon: '/copy.svg',
```

Comment:
Don't we want to rename this and `server.svg` to the exact menu? like `node.svg` and `registries.svg`?

Reply:
`copy.svg` is used elsewhere, so we could keep 2 copies of it one for the sidebar (`registries.svg`) and one for everything else (`copy.svg`).

Reply:
Ahh not the right implementation, right now its okey. Approved.